firedrake/slate/static_condensation/hybridization.py

```diff
@@ -1,21 +1,19 @@
-"""This module provides custom python preconditioners utilizing
-the Slate language.
-"""
 import ufl
 import numpy as np
 
-from firedrake.preconditioners import PCBase
+from firedrake.slate.static_condensation.sc_base import SCBase
 from firedrake.matrix_free.operators import ImplicitMatrixContext
 from firedrake.petsc import PETSc
 from firedrake.parloops import par_loop, READ, INC
 from firedrake.slate.slate import Tensor, AssembledVector
 from pyop2.profiling import timed_region, timed_function
 from pyop2.utils import as_tuple
 
+
 __all__ = ['HybridizationPC']
 
 
-class HybridizationPC(PCBase):
+class HybridizationPC(SCBase):
```

Comment:
The only reason I have not made `HybridizationPC` a subclass of `SCPC` is that I have not figured out how to transform the two-field UFL problem into a complete 3-field hybrid problem. I mentioned this in the Slack channel (general chat).